styles/Navbar.module.css

```diff
@@ -0,0 +1,93 @@
+.header{
+    border-bottom: 1px solid #E2E8F0;
+}
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 1rem 1.5rem;
+}
+
+.none {
+    display: none;
+}
+
+.bar {
+    display: block;
+    width: 35px;
+    height: 3px;
+    margin: 5px auto;
+    -webkit-transition: all 0.3s ease-in-out;
+    transition: all 0.3s ease-in-out;
+    background-color: #222222;
+}
+
+.navmenu {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.navitem {
+    margin-left: 5rem;
+}
+
+.navlink{
+    font-size: 2rem;
+    font-weight: 400;
```

Comment:
on the figma, it looks like this should be 18px and a weight of 500. Regardless,  can we go ahead and make a label-large class in global.css with the correct font-size's and font-weights, and then use that. Effectively, once everything is defined in global.css, in this file underneath .navlink, all you would have to do is say 
`.navlink {
  composes: label-large from '../global.css';`

In fact, can you update the global.css with all the global parameters we will be using. So the colors and the different font sizes that are defined in Figma.